src/Service/SearchIndex/IndexService/IndexHandler/AbstractIndexHandler.php

```diff
@@ -53,8 +53,18 @@ public function updateMapping(
             $this->doUpdateMapping($context);
         } catch (Exception $e) {
             $this->logger->info($e);
+
             //try recreating index
-            $this->reindexMapping($context, $mappingProperties);
+            try {
+                $this->reindexMapping($context, $mappingProperties);
+            } catch (Exception $e) {
+                if (!$forceCreateIndex) {
+                    $this->logger->warn('Retrying with force creating index because reindexing failed due to following error: ' . $e);
```

Comment:
my bad
https://github.com/pimcore/generic-data-index-bundle/pull/361/files#r2472162046

```suggestion
                    $this->logger->warning('Retrying with force creating index because reindexing failed due to following error: ' . $e);
```